src/libsystemd/sd-event/sd-event.c

```diff
@@ -5153,6 +5153,27 @@ _public_ int sd_event_source_is_ratelimited(sd_event_source *s) {
         return s->ratelimited;
 }
 
+_public_ int sd_event_source_leave_ratelimit(sd_event_source *s) {
```

Comment:
Maybe better to add a brief test about this?

Reply:
Still I'd like to have a simple test case for this.

Reply:
yupp, working on it